src/CalcViewModel/DataLoaders/CurrencyDataLoader.cpp

```diff
@@ -644,6 +644,18 @@ void CurrencyDataLoader::GuaranteeSelectedUnits()
             isConversionTargetSet = true;
         }
     }
+
+    // If still not set for either source or target, just select the first currency in the list
```

Comment:
This change doesn't seem related to switching endpoints.  We should probably pull this specific change out to a separate PR.

Reply:
Switching the data set to one that does not contain the expected default values (in this case the dollar and the euro) results in the UI not selecting anything by default in the currency converter mode. Since this PR is changing the data set, I think it should be included in this PR.

Reply:
Fair enough.  Keep as-is then.